src/dataflow-types/src/logging.rs

```diff
@@ -108,8 +108,14 @@ impl LogVariant {
             LogVariant::Timely(TimelyLog::Addresses) => RelationDesc::empty()
                 .with_column("id", ScalarType::Int64.nullable(false))
                 .with_column("worker", ScalarType::Int64.nullable(false))
-                .with_column("slot", ScalarType::Int64.nullable(false))
-                .with_column("value", ScalarType::Int64.nullable(false))
+                .with_column(
+                    "address",
+                    ScalarType::List {
+                        element_type: Box::new(ScalarType::Int64),
+                        custom_oid: None,
+                    }
+                    .nullable(false),
+                )
                 .with_key(vec![0, 1, 2]),
```

Comment:
Since the "slot" column is gone, the key of this table should now be `vec![0,1]`.

Reply:
Thanks, that's a good catch!